### ISSUE TITLE
[BUG] Flowkeeper doesn't start on GNOME

### ISSUE THREAD

**Issue by co-stig** [flowkeeper-org/fk-desktop#16]
On Debian 12 (6.1.0-13), Wayland, GNOME 43.6. Flowkeeper 0.3.3 installed via DEB, fails to start with an error:

```
flowkeeper: symbol lookup error: /usr/local/lib/flowkeeper/_internal/libQt6WaylandClient.so.6: undefined symbol: wl_proxy_marshal_flags
```

On the same system, when logged into an Xorg session, the error is different:

```
qt.qpa.plugin: From 6.5.0, xcb-cursor0 or libxcb-cursor0 is needed to load the Qt xcb platform plugin.
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, linuxfb, offscreen, wayland, xcb, vkkhrdisplay, vnc, minimal, minimalegl, wayland-egl.
```

Exactly the same issue happens on 
- Ubuntu 22.04.3 / GNOME 42.9 Wayland and Xorg;
- Fedora 39 Workstation / GNOME 45.0 Wayland and Xorg;
- RHEL 9.3 / GNOME 40.4.0 Wayland and Xorg;

**Comment by co-stig** [comment]
See https://bugreports.qt.io/browse/QTBUG-114635, one suggestion is to add `-platform xcb` to the command line -- to try on Wayland.

**Comment by co-stig** [comment]
Fixed by building Flowkeeper against Ubuntu 22.04 instead of 20.04.
